src/FunderMaps.Portal/appsettings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "Jwt": {
+    "Issuer": null,
+    "Audience": null,
+    "TokenValidity": 0,
+    "SignatureKey": null
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+  "Email": {
+    "SmtpServer": null,
+    "SmtpPort": 465,
+    "SmtpUsername": null,
+    "SmtpPassword": null,
+    "SmtpTls": true,
+    "DefaultSenderName": null,
+    "DefaultSenderAddress": null
+  },
+  "ConnectionStrings": {
+    "FunderMapsConnection": null
+  },
+  "BlobStorage": {
+    "AccessKey": null,
```

Comment:
This used to be AccesKey (with one s). I'm not completely certain that this typo has been resolved already. Has this been checked, since most of the times we use a nullservice?

Reply:
Yup, document upload and download does work